modules/source-bookshelf/lib/process_sort.js

```diff
@@ -3,11 +3,12 @@ const _ = require('lodash');
 // TODO: investigate how to deal with express's query parser
 // converting + into a space.
 function isAscending (key) {
-  return key[0] === '+' || key[0] === ' ';
+  return key[0] === '+' || key[0] === '-';
 }
 
 module.exports = function (model, query, sortBy) {
   var validFields = model.fields;
+  console.log(model.fields);
```

Comment:
Also, `source.fields()` is supposed to provide a consistent interface to the underlying fields listing no matter what the ORM.